casper/src/test/scala/coop/rchain/casper/batch2/EstimatorHelperTest.scala

```diff
@@ -1,150 +1,149 @@
-          _  <- n4.syncWith(n1, n2, n3)
-
-          b7  <- n3.addBlock(produceDeploys(3))
-          b8  <- n4.addBlock(produceDeploys(5))
-          b9  <- n3.addBlock(consumeDeploys(5))
-          b10 <- n4.addBlock(produceDeploys(4))
-
-          _   <- n4.syncWith(n3)
-          dag <- n4.blockDagStorage.getRepresentation
-
-          _ <- conflicts[Task](b2, b3, dag) shouldBeF false
-          // b4 conflicts with b5 directly
-          _ <- conflicts[Task](b4, b5, dag) shouldBeF true
-          _ <- conflicts[Task](b6, b6, dag) shouldBeF false
-          _ <- conflicts[Task](b6, b9, dag) shouldBeF false
-          _ <- conflicts[Task](b7, b8, dag) shouldBeF false
-          _ <- conflicts[Task](b7, b10, dag) shouldBeF false
-          // b8 conflicts with b9 directly
-          _ <- conflicts[Task](b8, b9, dag) shouldBeF true
-          // b9 conflicts indirectly with b10 due to the fact that b8 is the latter's ancestor
-          _ <- conflicts[Task](b9, b10, dag) shouldBeF true
-        } yield ()
-    }
-  }
-
-  "Blocks" should "conflict if their deploys conflict in the deployLog" in withStorage {
-    implicit blockStore => implicit blockDagStorage =>
-      testConflict[Task] { deploy =>
-        deploy.copy(
-          deployLog = List(
-            produce(ByteString.copyFromUtf8(channelsHash))
-          )
-        )
-      } { deploy =>
-        deploy.copy(deployLog = List(consume(ByteString.copyFromUtf8(channelsHash))))
-      }
-  }
-
-  private def testConflict[F[_]: Sync: BlockStore: IndexedBlockDagStorage: Time: Log: Span](
-      deployMod1: ProcessedDeploy => ProcessedDeploy
-  )(
-      deployMod2: ProcessedDeploy => ProcessedDeploy
-  ): F[Unit] = {
-    implicit def blockMessageToBlockMetadata(b: BlockMessage): BlockMetadata =
-      BlockMetadata.fromBlock(b, invalid = false)
-
-    for {
-      genesis <- createGenesis[F]()
-      deployA <- basicProcessedDeploy[F](1).map(deployMod1)
-      a       <- createBlock[F](Seq(genesis.blockHash), genesis, deploys = Seq(deployA))
-      deployB <- basicProcessedDeploy[F](2).map(deployMod2)
-      b       <- createBlock[F](Seq(genesis.blockHash), genesis, deploys = Seq(deployB))
-      dag     <- BlockDagStorage[F].getRepresentation
-
-      _              <- conflicts[F](a, b, dag) shouldBeF true
-      nonconflicting <- EstimatorHelper.chooseNonConflicting(Seq(a, b).map(_.blockHash), dag)
-      _              = assert(nonconflicting == List[BlockMetadata](a))
-    } yield ()
-  }
-
-  private def produce(channelsHash: ByteString) =
-    ProduceEvent(
-      channelsHash,
-      hash = ByteString.copyFromUtf8("Asdfg753213fdsadfueias9fje35mv43"),
-      persistent = false,
-      timesRepeated = 0
-    )
-
-  private def consume(channelsHash: ByteString) =
-    ConsumeEvent(
-      List(channelsHash),
-      hash = ByteString.copyFromUtf8("Asdfg753213fdsadfueias9fje35mv43"),
-      persistent = false
-    )
-}
+//package coop.rchain.casper.batch2
```

Comment:
Maybe it's better to remove this completely, I'm not sure if this code is any useful now.